README.md

```diff
@@ -40,93 +40,72 @@
+[ESM](https://dev.to/iggredible/what-the-heck-are-cjs-amd-umd-and-esm-ikm),
+[CommonJS](https://www.typescriptlang.org/docs/handbook/2/modules.html#commonjs).
   
-
-### IIFE
-See how it's done in [this example](https://github.com/process-analytics/bpmn-visualization-examples/blob/master/examples/01-getting-started).
-
-
-### ESM
-🚀 You can find an integration example of Vanilla Typescript with rollup, [here](https://github.com/process-analytics/bpmn-visualization-examples/tree/master/examples/projects/typescript-vanilla-with-rollup).
+Here are examples per module: \
+[bpmn-visualization (IIFE)](https://github.com/process-analytics/bpmn-visualization-examples/blob/master/examples/01-getting-started) \
+[bpmn-visualization (ESM); Typescript + Rollup](https://github.com/process-analytics/bpmn-visualization-examples/tree/master/examples/projects/typescript-vanilla-with-rollup)
 
 ⚒️ If you want to configure yourself:
 * Install the dependency in your **package.json** file:
 ```shell script
 npm i bpmn-visualization
 ```
 
-* Load necessary scripts 
+* Load bpmn-visualization script (replace _X._Y._Z by the recent version)
 ```html
-    <!-- load global settings -->
-    <script src="./static/js/configureMxGraphGlobals.js"></script>
-    <!-- load mxGraph client library -->
-    <script src="./static/js/mxClient.min.js"></script>
-    <!-- load BPMN Visualization library -->
-    <script src="index.esm.js" type="module"></script>
+    <script src="https://unpkg.com/bpmn-visualization@_X._Y._Z/dist/bpmn-visualization.js"></script>
 ```
 * Define your BPMN content using one of the following ways:
   * Copy/Paste directly the XML content in a variable
   * Load it from a url, like this [example](https://github.com/process-analytics/bpmn-visualization-examples/blob/master/examples/load-remote-bpmn-diagrams/index.html)
   * Load from your computer, like the [demo example](https://github.com/process-analytics/bpmn-visualization-js/blob/master/src/demo/index.ts)
 ```javascript
-    const bpmnContent = ``; // your BPMN 2.0 XML content
+    function bpmnDiagram() {
+      return  ``; // your BPMN 2.0 XML content
+    }
 ```
-* Define the mxGraph container
+* Define the bpmn diagram container
 ```html
-    <div id="graph"></div>
-```
-* Initialize BpmnVisualization from the container
-```javascript
-    const container = 'graph';
-    console.log(`Initializing BpmnVisualization with container '${container}'...`);
-    const bpmnVisualization = new BpmnVisualization(window.document.getElementById(container));  
+    <div id="bpmn-container"></div>
 ```
-* Load the bpmn content from BpmnVisualization
+* Initialize BpmnVisualization and load the diagram
 ```javascript
-    console.log('Loading bpmn....');
-    bpmnVisualization.load(bpmnContent);
-    console.log('BPMN loaded');
+    const bpmnVisualization = new bpmnvisu.BpmnVisualization(document.getElementById('bpmn-container'));
+    bpmnVisualization.load(bpmnDiagram());
```

Comment:
As I said before I prefer to let as it was in order to have the code the more simple as possible.
If the user want more complexity, he can see the examples.

```suggestion
    bpmnVisualization.load(bpmnContent);
```